dev/Expander/Expander.xaml

```diff
@@ -19,10 +20,21 @@
                Without this override, the narrator user that focuses the expander on a touch screen will see that pressing "Tab"
                doesn't work how they would expect.-->
         <Setter Property="IsTabStop" Value="False"/>
+        <Setter Property="Background" Value="{ThemeResource ExpanderBackground}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
```

Comment:
nit: I think this should be moved to the bottom under `VerticalContentAlignment` for consistent formatting